CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.2.0
+
+- Bump Kotlin and Gradle versions (https://outsystemsrd.atlassian.net/browse/RMET-3887)
+
```

Comment:
Question, should this change alone warrant a minor version upgrade?

Reply:
given we're upgrading kotlin and gradle, we usually increase the minors